Assets/Scripts/ProjController.cs

```diff
@@ -0,0 +1,66 @@
+using UnityEngine;
+
+public class ProjController : MonoBehaviour
+{
+    [SerializeField]
+    private int projDamage = 100;
```

Comment:
I don't think there's much benefit to giving these instance variables default values because they will be set in the Inspector.

Reply:
Fair enough, but also what are the drawbacks of having default values?

Reply:
There's not really a drawback in this case. I usually avoid putting in default values due to the behavior described in [this post](https://stackoverflow.com/a/51558490), but the functionality described there doesn't really apply here. I feel like they're kind of unnecessary, but it might be good to have baseline projectile values as defaults. There's no real need to remove the default values.